### PR TITLE
Hw 1 Hoi-Ying Mak

### PR DESCRIPTION
**Pull request by makhy**
The easy parts are forking and cloning the repo and opening it in my RStudio. I’ve done that once when I configured Git as part of the software setup. Then, I referred to the R cheatsheet and the R markdown video to figure out how to use the markdown. The hard part is I don’t know where to type these markdown codes. There was a second window popped up for R markdown, but I couldn’t find a place to type there. I also had a problem with embedding an image in the readme. Adding the other features eg. Links, bold are easy though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # hw01
 Repository for homework 1
+# Hoi Ying's biography
+###### I am a MAPSS student at Uchicago. I like cooking and ballet dancing. My favorite holiday is **April Fools**.
+[link](https://en.wikipedia.org/wiki/April_Fools%27_Day)
+* The cities I like the best
+   + Chicago
+      - when it doesn't snow
+   + Tokyo
+1. My favorite GOT characters
```
Comment (gmvelez) [inline README.md:10]: Minor thing, but since this part only has 1, it doesn't need to be a numbered list

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
 # hw01
 Repository for homework 1
+# Hoi Ying's biography
+###### I am a MAPSS student at Uchicago. I like cooking and ballet dancing. My favorite holiday is **April Fools**.
+[link](https://en.wikipedia.org/wiki/April_Fools%27_Day)
```
Comment (gmvelez) [inline README.md:5]: Good use of different formatting and the link.

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for Hoi-Ying Mak
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |    x       |              |            |

I left some comments on your document, but overall a good job since you used various formatting options, lists, a link, and a photo).  As one minor note, make sure to hit enter twice when you want to have a line break in an R Markdown document.

**Review by sternullo15 — COMMENTED** [review]
Coding Style: Excellent
Coding Strategy: Excellent
Presentation: Satisfactory
Achievement, creativity: Excellent
Ease of Access: Excellent

Your bio was great and very creative! You did a great job including all of the extra pieces we were assigned to add. I think there were a few items, which were already noted, that could have been a bit tidier, particularly the lists and the way you embedded the photo and the link (the phrase "April Fools" might have been hyperlinked instead!).

**Comment by yeseulbyeon** [comment]
Coding Style: Satisfactory
Coding Strategy: Excellent
Presentation: Excellent
Achievement, creativity: Excellent
Ease of Access: Excellent

I liked that you used a variety of text formatting styles. I also think the code could have been a bit tidier -- one suggestion I'd add is putting the caption of your picture _inside_ the closed brackets of .

**Pull request closed without merging**